test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -76,13 +80,19 @@ string TestFunctionCall::format(string const& _linePrefix, bool const _renderRes
 		{
 			bytes output = m_call.expectations.rawBytes();
 			bool const isFailure = m_call.expectations.failure;
-			result = isFailure ? failure : formatBytesParameters(output, m_call.expectations.result);
+			result = isFailure
+				? failure
+				: formatRawParameters(m_call.expectations.result);
 		}
 		else
 		{
 			bytes output = m_rawBytes;
 			bool const isFailure = m_failure;
-			result = isFailure ? failure : formatBytesParameters(output, m_call.expectations.result);
+			result = isFailure
+				? failure
+				: matchesExpectation()
```

Comment:
This is an important change that was needed to get the update in isoltest right if `left()` or `right()` was used to define the expected result.